### PR TITLE
Fix typos and resource names in walkthrough e2e logs

### PR DESCRIPTION
**Pull request by pmorie**
Pretty much what it sounds like.

### CODE REVIEW

**Comment by duglin** [comment]
LGTM

File: test/e2e/walkthrough.go [hunk 1]
```diff
@@ -68,17 +68,17 @@ var _ = framework.ServiceCatalogDescribe("walkthrough", func() {
 		)
 
 		//Broker and ServiceClass should become ready
-		By("Make sure the named Broker not exist before create")
+		By("Make sure the named ServiceBroker not exist before create")
```
Comment (vaikas-google) [inline test/e2e/walkthrough.go]: ServiceBroker does not exist?

Comment (pmorie) [reply test/e2e/walkthrough.go]: nice find

**Pull request merged**